### ISSUE TITLE
The French Muséum national d’histoire naturelle vascular plant herbarium collection dataset

### ISSUE THREAD

**Issue by rdmpage** [gbif/data-mobilization#57]
GBIF has this already but I just want to flag this data paper that describes the dataset:

Le Bras, G., Pignal, M., Jeanson, M. L., Muller, S., Aupic, C., Carré, B., … Haevermans, T. (2017, February 14). The French Muséum national d’histoire naturelle vascular plant herbarium collection dataset. Scientific Data. Springer Nature. https://doi.org/10.1038/sdata.2017.16

This paper cites the GBIF dataset DOI http://dx.doi.org/10.15468/nc6rxy  but the GBIF page doesn't cite the data paper, nor provide citing paper as an option. And can GBIF automatically discover that the dataset is cited by this paper?

I've raised this here only to flag the issue of monitoring duplications of data appears and linking those to existing GBIF datasets, as well as identifying new datasets to add.

**Comment by dschigel** [comment]
Hi @rdmpage rdmage, this is a much discussed functionality, and there are several implementation options on how to display the derived publication at the dataset pages where one or more records contributed to the download. A few recently discussed options include Google Scholar API and DLI https://dliservice-prototype-dli.d4science.org/#/ and your e-mails. There is not much to scope - we know how it should look like, but the implementation is likely to follow the launch of new portal, correct @timrobertson100?

**Comment by rdmpage** [comment]
@dschigel Yep, I was just flagging it for my own interest really. A "use case", if you will.

**Comment by kbraak** [comment]
@rdmpage thanks for raising this issue. It prompted me to:

1. Add two related FAQs to the IPT resource metadata how-to page:
    1. [Q. Can I add a link to the dataset's related data paper?](https://github.com/gbif/ipt/wiki/resourceMetadata#q-can-i-add-a-link-to-the-datasets-related-data-paper)
    2. [Q. GBIF assigned my dataset a DOI - can I add this to my metadata?](https://github.com/gbif/ipt/wiki/resourceMetadata#q-gbif-assigned-my-dataset-a-doi---can-i-add-this-to-my-metadata)
2. Contact the publisher with instructions to update their resource metadata in line with the above FAQs.

For activity on linking datasets to publications you can monitor the following issues:
* https://github.com/gbif/portal16/issues/275
* https://github.com/gbif/portal16/issues/307
* https://github.com/gbif/gbif-doi/issues/2

**Comment by dschigel** [comment]
Sorry, correction - just learnt from Daniel that there is no such thing as Google Scholar API.

**Comment by ahahn-gbif** [comment]
@dschigel could you check whether this issue still needs to be open here or is sufficiently covered by the linked issues from Kyle? It may be that your comment is better placed under one of those, I would not want to keep it open under the "data mobilization" project.

**Comment by dschigel** [comment]
Agreed to move out from data mob, but please keep open - we just had the case with West of Urals collection where I want to test exactly Rod's suggestion of datasets citing "their" papers. In my opinion it should work both ways.
